appveyor.yml

```diff
@@ -13,33 +13,36 @@ environment:
 
 cache:
   - composer.phar
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\php -> appveyor.yml
+  - c:\ProgramData\chocolatey\bin -> appveyor.yml
+  - c:\ProgramData\chocolatey\lib -> appveyor.yml
+  - c:\php -> appveyor.yml
 
 init:
-  - SET PATH=C:\Program Files\OpenSSL;C:\tools\php;%PATH%
+  - SET PATH=c:\Program Files\OpenSSL;c:\tools\php;%PATH%
   - SET COMPOSER_NO_INTERACTION=1
   - SET PHP=1
   - SET ANSICON=121x90 (121x90)
 
 install:
-  - IF EXIST C:\tools\php (SET PHP=0)
-  - ps: appveyor-retry cinst --params '""/InstallDir:C:\tools\php""' --ignore-checksums -y php --version ((choco search php --exact --all-versions -r | select-string -pattern $env:php_ver_target | sort { [version]($_ -split '\|' | select -last 1) } -Descending | Select-Object -first 1) -replace '[php|]','')
-  - cd C:\tools\php
+  - IF EXIST c:\tools\php (SET PHP=0)
+  - ps: choco search php --exact --all-versions -r
+  - ps: echo ((choco search php --exact --all-versions -r | select-string -pattern $env:php_ver_target | sort { [version]($_ -split '\|' | select -last 1) } -Descending | Select-Object -first 1) -replace '[php|]','')
```

Comment:
I'd prefer to keep these two lines, to debug what's going wrong next time builds on AppVeyor crashes with no obvious reason.